tests/examples/step-40.diff

```diff
@@ -17,11 +17,15 @@
 >                               solver_control.last_step(),
 >                               8,
 >                               15);
-634c636
+643c645
 <     const unsigned int n_cycles = 8;
 ---
 >     const unsigned int n_cycles = 3;
-662c664
+659c661
+<         output_results(cycle);
```

Comment:
I thought about this some more and I think we should keep all calls to graphical output.